### PR TITLE
Disabled Insecure request warning after login

### LINKED ISSUE #319: Suppress insecure request warnings when setting skip_tls to True
## WHY
Why is this change being made?
Currently when someone logs in with `skip_tls=True` you will receive a warning like this
```
warnings.warn(
/opt/app-root/lib64/python3.8/site-packages/urllib3/connectionpool.py:1045: InsecureRequestWarning: Unverified HTTPS request is being made to host <HOST>. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
```
This is fine you should be warned at least once but every other action such as `cluster.up()` will produce this warning.
Running `cluster.wait_ready()` will result in your console output being flooded with this warning while waiting for the cluster to be ready.
## WHAT
What is being asked for?
We should only allow this warning to show upon logging in.
## HOW
Suggestions for how this may be solved. [Optional]
We can pass `urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)` just before the successful login message
## TESTS
List of related tests
## DONE
Bullet point items for what should be completed

### PR DESCRIPTION
**Pull request by Bobbins228**
# Issue link

Closes #319 
# What changes have been made

Disabled Insecure request warning after login when setting `skip_tls=True`
# Verification steps

Following a guided demo
Login with `skip_tls=True`
You should get a warning message similar to...
```
warnings.warn(
/opt/app-root/lib64/python3.8/site-packages/urllib3/connectionpool.py:1045: InsecureRequestWarning: Unverified HTTPS request is being made to host <HOST>. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
```
Follow the rest of the guided demo and you shouldn't see the warning again.
## Checks
- [x] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [x] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

File: src/codeflare_sdk/cluster/auth.py [hunk 1]
```diff
@@ -112,6 +113,7 @@ def login(self) -> str:
             api_client = client.ApiClient(configuration)
             client.AuthenticationApi(api_client).get_api_group()
             config_path = None
+            urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```
Comment (KPostOffice) [inline src/codeflare_sdk/cluster/auth.py:116]: Could you do this on line 112, and just log that it's being done?

**Review by KPostOffice — APPROVED** [review]
I am going to add an issue for replacing prints with the python `logger` library.

**Review by MichaelClifford — APPROVED** [review]
LGTM

**Pull request merged**